src/Mail/exceptions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (http://nette.org)
+ * Copyright (c) 2004 David Grudl (http://davidgrudl.com)
+ */
+
+namespace Nette\Mail;
+
+use Nette;
+
+
+/**
+ * Send failed exception.
+ *
+ * @author Jan Dvořák
+ */
+class SendFailedException extends Nette\InvalidStateException
+{
+}
+
+
+/**
+ * SMTP mailer exception.
+ *
+ * @author     David Grudl
+ */
+class SmtpException extends SendFailedException
```

Comment:
Well, když teď nad tím tak přemýšlím, tak už chápu proč se to snaží udělat takhle... protože `SendFailedException` může pak házet i ten sendmail...